android/src/test/java/com/fidelreactlibrary/fakes/CountryAdapterStub.java

```diff
@@ -10,6 +12,7 @@
 public final class CountryAdapterStub implements CountryAdapter {
 
     public Country countryToReturn;
```

Comment:
This property is not used anymore; I think it can be removed. Please check if `countryIntegerReceived` can be removed too.